### ISSUE TITLE
occ duplicates:find-all get "User context required for this operation"

### ISSUE THREAD

**Issue by fwilly** [eldertek/duplicatefinder#110]
If I call
```bash
root@cloud02:/var/www/nextcloud/.CLOUD_DATA/first.user/files# sudo -u www-data ../../../nextcloud-11.0.1/occ -v duplicates:find-all -u first.user --path="Photos"
Start searching files for first.user in path /first.user/files/Photos
Scanning /first.user/files/Photos/iPhone_SE2/2021/02/21-02-11 20-59-26 0694.jpg
An error occurred during scanning: User context required for this operation
Duplicates for user "first.user" are:
```
What is the problem?

**Comment by Langelus** [comment]
Seeing the exact same error.

**Comment by eldertek** [comment]
Fixed in v1.5.0
